src/components/common/applicant-info-1.vue

```diff
@@ -440,10 +443,45 @@ export default class ApplicantInfo1 extends Mixins(ActionMixin) {
   ]
   showAddressMenu: boolean = false
 
-  mounted () {
+  async mounted () {
     // add event listener when this component is mounted
     // eg, when user continues to send for review
     this.$el.addEventListener('keydown', this.handleKeydown)
+
+    // proceed only if we have any roles (ie, logged in), and not staff
+    if (this.getKeycloakRoles.length > 0 && !this.isRoleStaff) {
+      // pre-populate submitting party name
+      const userInfo = await AuthServices.fetchUserInfo().catch(e => null)
+      if (userInfo) {
+        this.setApplicantDetails([
+          { name: 'firstName', value: userInfo.firstname || '' },
+          { name: 'lastName', value: userInfo.lastname || '' }
+        ])
+      }
+
+      // pre-populate submitting party address
+      const currentAccount = sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+      if (currentAccount) {
+        const accountId = JSON.parse(currentAccount)?.id
+        if (accountId) {
+          const orgInfo = await AuthServices.fetchOrgInfo(accountId).catch(e => null)
+          if (orgInfo) {
+            const mailingAddress = orgInfo.mailingAddress
+            if (mailingAddress.street) {
+              this.setApplicantDetails([
+                { name: 'addrLine1', value: mailingAddress.street || '' },
+                { name: 'addrLine2', value: mailingAddress.streetAdditional || '' },
+                { name: 'addrLine3', value: '' },
+                { name: 'city', value: mailingAddress.city || '' },
+                { name: 'stateProvinceCd', value: mailingAddress.region || '' },
+                { name: 'countryTypeCd', value: mailingAddress.country || '' },
+                { name: 'postalCd', value: mailingAddress.postalCode || '' }
+              ])
+            }
+          }
+        }
+      }
+    }
```

Comment:
Just curious, since we are making the request in this component vs when we load the app or via a login etc. Will this request be made each time the user navigates through the tabs or does `applicant-info-1` only mount once?

Reply:
Yes, this component in only mounted once.